raphtory-graphql/src/server.rs

```diff
@@ -299,22 +302,25 @@ impl RaphtoryServer {
         };
 
         dotenv().ok();
-        println!("Loading env");
-        let client_id_str = env::var("CLIENT_ID").expect("CLIENT_ID not set");
-        let client_secret_str = env::var("CLIENT_SECRET").expect("CLIENT_SECRET not set");
-        let tenant_id_str = env::var("TENANT_ID").expect("TENANT_ID not set");
+        let client_id = self.configs.auth.client_id.expect("No client id provided");
```

Comment:
I think all these expects we want to change to throw an error - the function should return a result